src/defense.cpp

```diff
@@ -1176,7 +1182,40 @@ std::vector<itype_id> caravan_items( caravan_category cat )
             return ret;
 
         case CARAVAN_MELEE:
-            item_list = item_group::items_from( "defense_caravan_melee" );
+            if( ( caravan_tech_level == 1 ) ){
```

Comment:
Don't need the parenthesis inside of the condition, can instead be `if( caravan_tech_level == 1 )` for each of these

Reply:
Fixed.